libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -2156,6 +2156,17 @@ bool AP_AHRS_NavEKF::have_ekf_logging(void) const
     return false;
 }
 
+//get the index of the active airspeed sensor, wrt the primary core
+int8_t AP_AHRS_NavEKF::get_active_airspeed_index() const
+{
+// we only have affinity for EKF3 as of now
+#if HAL_NAVEKF3_AVAILABLE
```

Comment:
this needs to check if EKF3 is the active EKF instance (EKF3 may be compiled in but not being used). You can use active_EKF_type() to determine which EKF is active

Reply:
fixed, thanks